examples/seldon_deployment/run.py

```diff
@@ -0,0 +1,209 @@
+#  Copyright (c) ZenML GmbH 2022. All Rights Reserved.
```

Comment:
The Readme for this example is still WIP?

Reply:
Yes, you're correct. Next on my list.